### PR TITLE
Move gamelab animation library to s3

### PR DESCRIPTION
**Pull request by ajpal**
Now that the spritelab manifest is in S3 (see https://github.com/code-dot-org/code-dot-org/pull/34790), we want to have the gamelab one there as well.

image
image
image
image

S3 Bucket:
image

## Links

- [jira](https://codedotorg.atlassian.net/browse/STAR-1167)

## Testing story

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Comment by ajpal** [comment]
@bethanyaconnor Should I also remove the spriteCostumeLibrary.json from our codebase? It doesn't look like we use it anymore, right?

**Comment by bethanyaconnor** [comment]
It's safe to remove `spriteCostumeLibrary.json`. Up to you if you want to do that in this PR or I can do that in a follow-up PR when I attempt to clean up all my code from the animation library translation project.

It's probably worth updating [the script](https://github.com/code-dot-org/code-dot-org/blob/a590c9471ab2003592870b7805753568ebe4223d/tools/scripts/ManifestBuilder.rb#L92) that actually creates these manifests to also upload the gamelab manifest. I'm planning on doing a clean up pass at that file probably next week so I'm happy to do that as well (there's no reason it writes to a file AND uploads to S3 for instance but that doesn't belong in this PR).

File: shared/middleware/animation_library_api.rb [hunk 1]
```diff
@@ -43,10 +43,11 @@ class AnimationLibraryApi < Sinatra::Base
   # Retrieve the manifest from S3. This will be extended to take in
```
Comment (bethanyaconnor) [inline shared/middleware/animation_library_api.rb:43]: Do you mind updating this comment? I'm still planning on adding locale to this function but we can remove the gamelab support piece.

**Review by bethanyaconnor — APPROVED** [review]
Thanks for doing this!

**Comment by ajpal** [comment]
> It's probably worth updating the script that actually creates these manifests to also upload the gamelab manifest. I'm planning on doing a clean up pass at that file probably next week so I'm happy to do that as well (there's no reason it writes to a file AND uploads to S3 for instance but that doesn't belong in this PR).

That's also part of https://codedotorg.atlassian.net/browse/STAR-1167, planning to tackle that tomorrow :)

**Pull request merged**